### PR TITLE
Approval view #87

### LINKED ISSUE #87: Approval View (Component)
Special view for the Approver to be able to approve/reject and comment on process models.

For more details see document on SharePoint:
`/Development/APs_Requirements/EBUSA_SWD_APs_29.04.18.pdf`

### PR DESCRIPTION
**Pull request by Graf-Carello**
resolves #87
This is the initial implementation of the approval page. Further To-Do is to implement the actual API calls to the specific endpoints to use real data instead of test data.

### CODE REVIEW

File: GUI-dev/src/app/pages/approval/approval.component.html
Comment (amarbajric) [inline GUI-dev/src/app/pages/approval/approval.component.html]: What should this do or be?

**Comment by amarbajric** [comment]
At first glance the idea is kinda acceptable. 
Nevertheless, there are some aspects which need refactoring and adaptations.

**1.**
This section should be the parent component (i.e. this is what an approver should see if he visits the tab `Approval Page`.)
screen shot 2018-06-04 at 15 26 16

This section should be something like the approval-details page (i.e. when the approver selects one process from the list illustrated in the picture above)
screen shot 2018-06-04 at 15 26 56

**ToDo:**
- Split the list of processes which need approval and the process-details section into parent and child component
  - parent component => list of processes which need approval (with a simple list which shows user who uploaded it, name of process, description)
  - child component => if you click on one of the processes (that need approval) in the list of the parent component, another view should load where the information of the process and the user is shown, and where all comments are shown and the commenting, approve and rejection section should be at the bottom

**NOTE:**
As the process store is not yet implemented, it is only required to do the tasks mentioned above. Logic and final implementation hast to wait until the backend is ready. Best case scenario would be however, if you just take some mockup data and test it with them (just look up some tuts. on how to create mockup data in Angular)

**Review by amarbajric — CHANGES_REQUESTED** [review]
Please fix and/or implement the requested changes which can be found in the comment above.
If done, push your changes to this branch and notify me.

**Comment by Graf-Carello** [comment]
I implemented your requested changes now and pushed them to my feature branch.

**Comment by amarbajric** [comment]
For me it is still incomplete.
To avoid redundancy in the comments, please check my previous request (comment) on how I thought about the implementation.
In short, I would suggest to have one page which lists all processes that require a review.
If the approver clicks on one of the processes in the list, the application switches the page to a details page where you ONLY see the one process and can comment, approve or reject (deny) the process. Besides, relevant information about the process should also be shown there.

Nevertheless, I will approve this PR BUT it is still incomplete. Thus, as soon as the backend is providing all the necessary endpoints, this issue needs to be implemented well.

**Review by amarbajric — APPROVED** [review]
Needs further implementation as soon as endpoints are available

**Pull request merged**
